specs/altair/validator.md

```diff
@@ -125,6 +126,14 @@ class SignedContributionAndProof(Container):
     signature: BLSSignature
 ```
 
+### `SyncCommitteeSigningData`
+
+```python
+class SyncCommitteeSigningData(Container):
+    slot: Slot
+    subcommittee_index: uint64
+```
```

Comment:
this works for me -- i'll note there are some `Container` definitions in the phase 0 validator guide that are scattered throughout the text and not collected at the top -- something we can eventually fix

Reply:
https://github.com/ethereum/eth2.0-specs/pull/2251